.github/workflows/runtests.yml

```diff
@@ -227,9 +227,7 @@ jobs:
   python-flint:
     needs: code-quality
 
-    # Currently wheels are available for python-flint only on Windows and OSX
-    # so we use OSX rather than Ubuntu for this job.
-    runs-on: macos-latest
+    runs-on: ubuntu-20.04
```

Comment:
Now python-flint has wheels for Linux so this can use Ubuntu like most other jobs.